lbccore/src/main/kotlin/studio/lunabee/compose/core/LbcTextSpec.kt

```diff
@@ -265,9 +285,11 @@ sealed class LbcTextSpec {
         }
 
         @StringRes
-        private fun getStringIdByName(context: Context, name: String): Int? {
-            val res: Resources = context.resources
-            return res.getIdentifier(name, "string", context.packageName).takeIf { id -> id != 0 }
-        }
+        private fun getStringIdByName(resources: Resources, name: String): Int? = resources
+            .getIdentifier(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/671f64af15532e26dedee78ab5a669a32942fdfe/lbccore/src/main/kotlin/studio/lunabee/compose/core/LbcTextSpec.kt#L289"></span>Use of this function is discouraged because resource reflection makes it harder to perform build optimizations and compile-time verification of code. It is much more efficient to retrieve resources by identifier (e.g. <code>R.foo.bar</code>) than by name (e.g. <code>getIdentifier("bar", "foo", null)</code>).</td>
    </tr>
  </tbody>
</table>